docs/Gopkg.toml.md

```diff
@@ -192,7 +192,8 @@ The following are the current available options:
 
 Out of an abundance of caution, dep non-optionally preserves files that may have legal significance.
 
-Pruning is disabled by default. It can be enabled by setting them to `true` at the root level.
+Pruning is enabled by default for `go-tests` and `unused-packages`. It can disabled by removing those fields from the `[prune]` table at the root level. Pruning can be enabled for `non-go` by setting it to true at the root level.
+
```

Comment:
Though the default behavior generating a `Gopkg.toml` during `init` is to add `go-tests` and `unused-packages`, I think the language here is referring to the default behavior during *processing* of a `Gopkg.toml` when no explicit `true`/`false` value is provided. That being said, the original text could still use some clarification.

Reply:
I agree with @jmank88 . How about:

> Pruning options are disabled by default. Generating a `Gopkg.toml` generate through `dep init` will enable `go-tests` and `unused-packages` prune options by default.

Reply:
Ah, ok. That makes sense. I will change the wording.